gui/package.json

```diff
@@ -15,6 +15,7 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@fontsource/roboto": "^5.0.8",
+    "@fontsource/roboto-mono": "^5.0.8",
```

Comment:
@gabrielpoca I added this package, but no idea how to actually use it for monospace fonts. any idea?